bucket/texworks.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "0.6.6",
+    "description": "A latex editor",
+    "homepage": "https://www.tug.org/texworks/",
+    "license": "GPL-2.0",
+    "url": "https://github.com/TeXworks/texworks/releases/download/release-0.6.6/TeXworks-win-0.6.6-202103111144-git_24442ac.zip",
+    "hash": "02409b2ef1a8c42c2704c40c70adac4da710be5242570fe06af30c0137ebb66e",
+    "bin": "TeXworks.exe",
+    "shortcuts": [
+        [
+            "TeXworks.exe",
+            "TeXworks"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/TeXworks/texworks/releases/latest",
+        "re": "TeXworks-win-(?<version>\\d\\.\\d\\.\\d)(?<full>.+).zip"
+    },
+    "autoupdate": {
+        "url": "https://github.com/TeXworks/texworks/releases/download/$matchVersion/TeXworks-win-$matchVersion$matchFull.zip"
```

Comment:
You need to fix autoupdate

"url": "https://github.com/TeXworks/texworks/releases/download/release-$version/TeXworks-win-$matchVersion$matchFull.zip"